wrangler-storage/src/main/java/io/cdap/wrangler/store/recipe/RecipeStore.java

```diff
@@ -119,6 +120,14 @@ private void saveRecipe(RecipeId recipeId, RecipeRow recipeRow, boolean failIfNo
     });
   }
 
+  public Recipe getRecipe(RecipeId recipeId) throws RecipeNotFoundException {
```

Comment:
Do we have retries setup in case of failures? It is important to set that for getRecipe as it could be used in pipelines also. TransactionRunner object that is created for recipe store can have retry configured if it is not done already.

Reply:
The injected transaction runner takes care of retrying sql queries, so we need not do this